src/components/modalWindow.js

```diff
@@ -0,0 +1,55 @@
+/*
+This is modal window component for logging out of application.
+*/
+
+import React from 'react';
+import {Button, Modal} from 'react-bootstrap';
+
+class LogoutModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleShow = this.handleShow.bind(this);
```

Comment:
if you declare a function like
```js
handleShow = () => {
// you code
}
```
no need to use this.handleShow = this.handleShow.bind(this);
since Es6 arrow functions does the binding